### ISSUE TITLE
How can I get probability for all labels when I get a reference result using detection model

### ISSUE THREAD

**Issue by harimkang** [openvinotoolkit/openvino#3949]
Hi,
I want to use detection model to obtain probability of all labels that can be classified by model through inference_engine. Is there any way?
the code is configured as below.
result = _net.infer({_input_blob_name: image})

Currently, return one of the most representative labels for each detection box and their confidence values, 
but I want to get a score for every label in each box.
Are there any possible examples of this? Or I'd like to ask you if you have any ideas.

Regards,
Harim

**Comment by jgespino** [comment]
Hi @harimkang 

If I understand correctly, you want to see all the inference results and their probability? If so, take a look at the [object_detection_demo](https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/python_demos/object_detection_demo). You can pass `--raw_output_message` and it will output the information below. If you want to see additional detections, try lowering the threshold with `-t <value>`, for example `-t 0.1`.

```
[ INFO ]  Class ID | Confidence | XMIN | YMIN | XMAX | YMAX
[ INFO ]    #45    |   0.954288 |   16 |  291 |  680 | 1020
[ INFO ]    #42    |   0.682098 |    7 |    0 |  113 |  451
[ INFO ]    #47    |   0.650986 |    1 |  411 |   34 |  555
[ INFO ]    #51    |   0.628112 |  218 |  795 |  271 |  898
[ INFO ]    #51    |   0.602704 |   97 |  867 |  249 |  916
[ INFO ]    #51    |   0.506650 |   21 |  405 |  165 |  484
```

This should point you in the right direction to implement in your application, hope this helps!

Regards,
Jesus

**Comment by harimkang** [comment]
@jgespino , thank you for your reply. 
However, I want to get more information added from the results.
For example, if the model has three classifiable labels.
I want to obtain probability for all three in each box.
Is there any way to do this part?
I want to come out as below or similar results. (only 1 box result with 3 label model)

[ INFO ] Class ID | Confidence | XMIN | YMIN | XMAX | YMAX
[ INFO ] #1 | 0.02446991205215454 | 16 | 291 | 680 | 1020
[ INFO ] #2 | 0.9752423167228699 | 16 | 291 | 680 | 1020
[ INFO ] #3 | 0.00028782570734620094 | 16 | 291 | 680 | 1020

When there are 3 labels as shown above, per box. I want to get all 3 probabilities. (total score sum is 1)
I'm asking because I can't see the contents of openvino's guide document even if I look it up.
(In other words, I want to get **all logits of the results** classified by the model.)
Thanks once again.

Regards,
Harim

**Comment by jgespino** [comment]
Hi @harimkang 

Understood, thanks for explaining. In that case, take a look at the [Image Classification Python* Sample](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_ie_bridges_python_sample_classification_sample_async_README.html) and run the sample by specifying the number of classes in your model using `-nt`. For example, with squeezenet model it would be `-nt 1000` as it has 1000 classes. This will print all the classes and it's probabilities. 

Regards,
Jesus

**Comment by harimkang** [comment]
Thank you for your reply.
